packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -6323,14 +6323,28 @@ abstract class RenderObjectElement extends Element {
   }
 }
 
-/// The element at the root of the tree.
+/// Deprecated. Will be removed in a future version of Flutter.
```

Comment:
You could also mention that it is no longer used by Flutter.

Reply:
Added.